examples/js/archive.js

```diff
@@ -127,28 +132,22 @@ function showImages(getUrl) {
   axios.get(url)
       .then((response) => {
         if (response.data.files && response.data.files.length != 0) {
-          const imageThumbnails = response.data.files.filter(file => file.source === 'derivative');
-          const fullResImages = response.data.files.filter(file => file.format === 'PNG' || file.format === 'JPEG');
+          // number of images (thumbnails and fullres files) in response data
```

Comment:
Let's move L135-138 into the constructor, and remove references to `count` from archive.js.